cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
should you use global-plugin-config as the name or something to make it clear to users that it only lists the global plugin config?